configs/webpack/common.js

```diff
@@ -131,6 +141,7 @@ module.exports = {
       SIMULATOR_HAS_CPYTHON: JSON.stringify(dependencies.cpython !== undefined),
       SIMULATOR_HAS_AMMO: JSON.stringify(dependencies.ammo !== undefined),
       SIMULATOR_LIBKIPR_C_DOCUMENTATION: JSON.stringify(libkiprCDocumentation),
+      SIMULATOR_I18N: JSON.stringify(i18n),
```

Comment:
Any way to include `i18.n.json` in a way that it gets picked up by `yarn watch`?

Reply:
We can watch for it, but not sure how to specify "as a part of the watch, re-read the JSON file".

Reply:
Hm yeah. I think it'd be nice to have when working on translations. We can create an issue for now